client/finality-grandpa/src/communication/gossip.rs

```diff
@@ -107,6 +107,13 @@ const CATCH_UP_PROCESS_TIMEOUT: Duration = Duration::from_secs(15);
 /// catch up request.
 const CATCH_UP_THRESHOLD: u64 = 2;
 
+const PROPAGATION_ALL: u32 = 4; //in rounds;
+const PROPAGATION_ALL_AUTHORITIES: u32 = 2; //in rounds;
+const PROPAGATION_SOME_NON_AUTHORITIES: u32 = 3; //in rounds;
+const ROUND_DURATION: u32 = 4; // measured in gossip durations
```

Comment:
Shouldn't this be 2 (prevote and precommit)? Or are the extra 2 gossip durations to account for network/gossip delays?